packages/vuetify/src/components/VTextarea/__tests__/VTextarea.spec.cy.tsx

```diff
@@ -27,8 +27,8 @@ describe('VTextarea', () => {
       .should(el => expect(el.outerHeight()).to.equal(56))
       .click()
       .type('Lorem ipsum dolor sit amet consectetur adipisicing elit. ')
-      .should(el => expect(el.outerHeight()).to.equal(78))
+      .should(el => expect(el.outerHeight()).to.equal(80))
```

Comment:
Input padding top/bottom has been adjusted to 16px by default, so:

- For one row, height (56px) = line-height (24px) + padding-top (16px) + padding-bottom (16px)
- For two rows,  height (80px) = line-height (24px) * 2 + padding-top (16px) + padding-bottom (16px)